android/app/build.gradle

```diff
@@ -81,7 +81,7 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'com.android.support.test:runner:1.0.1'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.1'
-    compile(name:'breez', ext:'aar')
```

Comment:
The change from `compile` to `implementation` is necessary because the deprecated `compile` was removed, `implementation` is the natural sucessor but `api` is also an option, IMHO `implementation` makes more sense.